Makefile

```diff
@@ -7,14 +7,14 @@ SUDO := $(shell if groups | grep -q docker; then echo ''; else echo 'sudo'; fi)
 default: build
 
 # Customize these variables
-COMPONENT_FILENAME?=eth_price_oracle.wasm
-SERVICE_CONFIG_FILE?=.docker/service.json
+COMPONENT_FILENAME?=evm_price_oracle.wasm
+SERVICE_URL?="http://127.0.0.1:9999/service.json"
```

Comment:
not sure if 127.0.0.1 or 0.0.0.0 or the docker ip. need to verify on MacOS to get the right answer

Reply:
cool will migrate to this then